core/src/Revolution/Processors/Resource/Trash/GetList.php

```diff
@@ -62,18 +62,21 @@ public function prepareQueryBeforeCount(xPDOQuery $c)
         // delete_document - thats perhaps not necessary, because all documents are already deleted
         // but we need the purge_deleted permission - for every single file
 
+        if ($deleted = $this->getDeleted()) {
+            $c->where(['modResource.id:IN' => $deleted]);
+        } else {
+            return;
```

Comment:
`prepareQueryBeforeCount()` must always return the xPDOQuery

Reply:
And what kind of where to put here, so as not to return anything? Will this one work?

`
$c->where(['modResource.id' => 0]);
`

Reply:
Although it's probably better like this:

`
$c->where(['modResource.id:NOT IN' => $deleted]);
`